.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Thank you for reporting a possible bug in nodejs-itoolkit.
+
+Please fill in as much of the template below as you can.
+
+Node.js version: output of `node -v`
+itoolkit version: output of `npm ls itoolkit`
+IBM i version: output of `uname -vr`
+XMLSERVICE version: output of `qsh cmd('/qsys.lib/qxmlserv.lib/xmlver.pgm')`
+
+If possible, please provide code that demonstrates the problem, keeping it as
+simple and free of external dependencies as you can.
+-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1.
+2. 
+3.
+4. 
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+* **Node.js version**:
+* **itoolkit version**:
+* **IBM i version**:
+* **XMLSERVICE version***:
```

Comment:
Please move these up before the bug description

Reply:
Fixed